requirements.txt

```diff
@@ -44,4 +44,5 @@ SQLAlchemy-Utils==0.38.3
 toml>=0.10.1
 twilio==7.15.0
 Unidecode==1.3.4
+validators==0.20.0
```

Comment:
Ohh, is this for pydantic specifically?

Reply:
This one we brought into fideslib because one of our users (left unmentioned here for confidentiality) wanted to be able to specify urls as partial urls (https://my-site* and things like this). FastAPI could handle this, but Pydantic's url validator couldn't. Using this validators package let us specify the url as a string for pydantic, and still validate it.